### PR TITLE
new feature: find and replace

### PR DESCRIPTION
**Pull request by ruffle1986**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -52,16 +52,25 @@
...
       var path = item.getAttribute(pathAttr);
-      var img = document.createElement('img');
-      img.src = url(Math.round(sample.width), path);
+      var src = url(Math.round(sample.width), path);
 
-      item.appendChild(img);
+      if (findAndReplace) {
+        var imgToBeReplaced = item.querySelector('img');
+        if (imgToBeReplaced) {
+          imgToBeReplaced.outerHTML = '<img src="' + src + '" />';
```
Comment (oroce) [inline index.js]: are you sure it's a good idea?

i would rather go with `imgToBeReplaced.src = src;`

Comment (ruffle1986) [inline index.js]: aw yes, you're totally right.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lazy-image-loader",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (oroce) [inline package.json]: next time we shouldnt include version bumps in pr pls

Comment (ruffle1986) [inline package.json]: sure. sorry.

**Pull request merged**